CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+- check-smart.rb: Removed duplicated smartctl runs
```

Comment:
These two should be under a sub header of `### Removed`

Reply:
I've updated the Changelog with Added, Removed and Changed sub headings